app/components/CapitalProjectsList/CapitalProjectsDrawer.tsx

```diff
@@ -0,0 +1,66 @@
+import { Box, Flex, Heading } from "@nycplanning/streetscape";
+import { Slide } from "@chakra-ui/transition";
+import { Agency, CapitalProject } from "~/gen";
+import { CapitalProjectsList } from "./CapitalProjectsList";
+import { useState } from "react";
+
+export interface CapitalProjectsDrawerProps {
+  capitalProjects: Array<CapitalProject>;
+  district: string;
+  agencies: Agency[];
+  children: React.ReactNode;
+}
+
+export const CapitalProjectsDrawer = ({
+  capitalProjects,
+  district,
+  agencies,
+  children,
+}: CapitalProjectsDrawerProps) => {
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  return (
+    <Slide direction="bottom" in={true} style={{ zIndex: 10 }}>
+      <Flex
+        padding={{ base: 3, lg: 4 }}
+        background={"white"}
+        direction={"column"}
+        borderTopLeftRadius={"base"}
+        borderTopRightRadius={"base"}
+        gap={4}
+      >
+        <Box
+          height={"4px"}
+          width={20}
+          backgroundColor={"gray.300"}
+          alignSelf={"center"}
+          role="button"
+          borderRadius={"2px"}
+          aria-label={
+            isExpanded
+              ? "Collapse project list panel"
+              : "Expand project list panel"
+          }
+          onClick={() => {
+            setIsExpanded(!isExpanded);
+          }}
+        />
+        <Heading color="gray.600" fontWeight={"bold"} fontSize={"lg"}>
+          {district}
+        </Heading>
+
+        <Flex
+          height={{ base: isExpanded ? "85vh" : "40vh", lg: "auto" }}
```

Comment:
There's a minor issue where the Pagination component shifts up on mobile when you expand the drawer. Changing `85vh` to `70vh` here seems to fix it but not sure if that breaks anything else. If you're reasonably confident that isn't going to cause any downstream effects, I think it's a good change to make now.

Reply:
I don't think we should rely on setting a different height value to fix this. We should fix the placement of the filter menu first and then have a standardized height for the expansion before setting that value. See below for what I meant with the filter menu peeking out from my earlier comment. 
<img width="410" alt="image" src="https://github.com/user-attachments/assets/9b2c5c72-7cea-4454-b1c0-244488e7e074">

If we continue to render the pagination footer as a child we pass to the drawer or accordion component, we could probably have the Flexbox determine the anchoring of pagination with something like "align-content: space-between" or something. But I strongly feel that this work should be part of a separate issue.

Reply:
Understood. As a solution for now that doesn't mess with the `vh` value, how about putting `marginTop={"auto"}` on the `pagination` Flex on line 49 in  city-council-districts.$cityCouncilDistrictId.capital-projects.tsx? That will make it anchor the the bottom of its parent.

Reply:
That works, I've updated 👍

Reply:
Hello 👋 , I noticed this behavior when integrating this project list component with the project detail panel. I believe I have a working solution. I'm writing some tests now and should have it ready for review later today.